scripts/lib/CIME/case/case.py

```diff
@@ -1733,10 +1733,13 @@ def create(self, casename, srcroot, compset_name, grid_name,
     def new_hash(self):
         """ Creates a hash
         """
+        print(sys.argv)
```

Comment:
I think you can get rid of these prints.

Reply:
@jgfouca Removed the print statements.